head.html

```diff
@@ -1,4 +1,6 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
-<script src="/scripts/aem.js" type="module"></script>
-<script src="/scripts/scripts.js" type="module"></script>
-<link rel="stylesheet" href="/styles/styles.css"/>
+<script src="./scripts/benchmark.js" type="module"></script>
```

Comment:
I'd skip the benchmarking for the prod version, and consider this like debug statements for now

Reply:
Yes, this script is not supposed to be included in production pages. It is an optional dependency.